repo/packages/H/history/0/marathon.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "history",
+  "cpus": 0.5,
+  "mem": 512,
+  "instances": 1,
+  "ports": [5055],
```

Comment:
To require the resource offer to contain this port, you'd also need to set the `requirePorts` field to `true`.